### PR TITLE
registry: Add Missing Rust Crates

### PR DESCRIPTION
**Pull request by hdost**
We recently cleaned up our README, and started referencing the Registry, but we want to make sure that the crates are still in the registry.

Relates https://github.com/open-telemetry/opentelemetry-rust/issues/1582

### CODE REVIEW

File: data/registry/exporter-rust-user-events-logs.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+language: rust
+tags:
+  - rust
+  - exporter
+  - linux
+  - user_events
+license: Apache 2.0 OR MIT
+description:
+  A Rust OpenTelemetry Log exporter for
+  https://docs.kernel.org/trace/user_events.html
```
Comment (svrnm) [inline data/registry/exporter-rust-user-events-logs.yml:12]:
```suggestion
  A Rust OpenTelemetry Log exporter for
  [Linux Kernel User Events](https://docs.kernel.org/trace/user_events.html)
```

You can use markdown in the description, so maybe something like this?

**Review by svrnm — APPROVED** [review]
Thanks, this is really great! Overall looks good to me

File: data/registry/exporter-rust-user-events-logs.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title: User Events Logs Exporter
```
Comment (lalitb) [inline data/registry/exporter-rust-user-events-logs.yml:1]: In the [kernel ](https://docs.kernel.org/trace/user_events.html) and [crate ](https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-user-events-logs) documentation, the term "User Events" is consistently referred to as user_events. We can stick with this notation.

File: data/registry/exporter-rust-user-events-logs.yml (hunk 1 above)
Comment (lalitb) [reply data/registry/exporter-rust-user-events-logs.yml:12]: nit - A Rust OpenTelemetry Log exporter for
	  [Linux Kernel user_events](https://docs.kernel.org/trace/user_events.html)

File: data/registry/tools-rust-aws.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+title: AWS Utilities
+registryType: utilities
+description: Utilities related to AWS including XRay Propagators.
```
[the remark this answers is not in the document]
Comment (hdost) [reply data/registry/tools-rust-aws.yml]: Added.

File: data/registry/exporter-rust-user-events-logs.yml (hunk 1 above)
Comment (hdost) [reply data/registry/exporter-rust-user-events-logs.yml]: Done.

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
@hdost you can fix the format issue either by running `npm run fix:format` or comment `/fix:format` on this PR

**Comment by hdost** [comment]
✅

**Review by theletterf — APPROVED** [review]

File: data/registry/exporter-rust-stackdriver.yml [hunk 4]
```diff
@@ -9,8 +9,9 @@ license: Apache 2.0 OR MIT
 description: A Rust OpenTelemetry exporter for Google StackDriver
 authors:
   - name: jacobkiesel
+  - name: OpenTelemetry Authors
```
Comment (cijothomas) [inline data/registry/exporter-rust-stackdriver.yml:12]: any reason why we still keep the prior author name?

**Review by cijothomas — APPROVED** [review]
thanks!

**Pull request merged**